### ISSUE TITLE
Plugin could not be activated because it triggered a fatal error.

### ISSUE THREAD

**Issue by Swiftloke** [bobmatyas/wp-block-ai-crawlers#23]
I realize this is a little on the vague side, but that's all I know; I get this upon installing your plugin. I ended up installing the ``Wp Robots Txt`` plugin instead and filled it in myself using https://raw.githubusercontent.com/ai-robots-txt/ai.robots.txt/main/robots.txt.

I'm running WordPress 5.6.3.

**Comment by bobmatyas** [comment]
Hi there -

I'm sorry for the trouble here. 

I tried to reproduce using WordPress v5.6.3 but I was unable to replicate the issue. Would you mind sharing your site Health Info so that I can look into this more? 

http://example.com/wp-admin/site-health.php?tab=debug

**Comment by bobmatyas** [comment]
👋 @Swiftloke -

I tracked down the source of this issue:

```
[24-May-2024 15:52:38 UTC] PHP Parse error: syntax error, unexpected end of file in /var/www/html/wp-content/plugins/block-ai-crawlers/inc/settings.php on line 136
```

It should be fixed in the latest (v1.3.3). 

Thanks for the report1
